src/standalone/Vegetation/canopy_boundary_fluxes.jl

```diff
@@ -140,6 +140,12 @@ function ClimaLSM.surface_air_density(
     return compute_ρ_sfc.(Ref(thermo_params), Ref(ts_in), T_canopy)
 end
 
+function make_update_boundary_vars(canopy::CanopyModel)
+    function update_boundary_vars!(p, Y, t)
+        canopy_boundary_fluxes!(p, canopy, canopy.radiation, canopy.atmos, Y, t)
```

Comment:
do we need a separate `canopy_boundary_fluxes!` function? I think we can just move that code into here and remove that function

Reply:
I think this may be nice to have still, because in the SoilCanopyModel, which will have it's own `update_boundary_vars!` function, we can call `canopy_boundary_fluxes!` instead of copying that code over?

Reply:
Good point! That sounds good